snapcraft/internal/elf.py

```diff
@@ -269,6 +273,8 @@ def wrapper(*args, **kwargs):
             # should eventually be removed once patchelf catches up.
             try:
                 elf_file_path = kwargs['elf_file_path']
+                logger.info('Failed to update {!r}. Retrying with '
+                            'different parameters.'.format(elf_file_path))
```

Comment:
Would it be helpful to actually say what we're doing, here (removing the `.note.go.buildid` section)?